spec/features/users/user_edit_spec.rb

```diff
@@ -12,7 +12,7 @@
     fill_in 'Email', with: ''
     click_button 'Update'
 
-    expect(page).to have_content 'Please fill in this field'
+    expect(page).to have_content 'Please fill in this field.'
```

Comment:
It is not. It's from our validate-field.js which adds this message as well as some accessibility-fu. I made a feeble attempt at using i18n to generate this string but the js file is processed by browserify which doesn't accept template langs. I opted to address this later.